docs/source/sft_trainer.mdx

```diff
@@ -98,6 +98,8 @@ trainer = SFTTrainer(
 trainer.train()
 ```
 
+Note that if you use a packed dataset and if you pass `max_steps` in the training arguments you will probably train your models for more than few epochs. 
```

Comment:
That's not really true, right? It could also be shorter - depends on the dataset and max_steps you choose :)